tools/dm/dm-upgrade.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Data Migration Upgrade
+summary: Lear how to upgrade a Data Migration version to a non-compatible version.
+category: tools
+---
+
+# Data Migration Upgrade
+
+This document introduces how to upgrade your Data Migration (DM) version to an incompatible version.
+
+Assuming that V-A, V-B, V-C are three DM versions in chronological order and they are not compatible with each other, you need to upgrade V-A to V-C. Upgrade-A-B means upgrading V-A to V-B and Upgrade-B-C means upgrading V-B to V-C.
+
+- If Upgrade-A-B overlaps Upgrade-B-C (e.g. different changes of a same configuration item), it is recommended to perform Upgrade-A-B to V-B and then perform Upgrade-B-C to V-C.
```

Comment:
Both are right, but it seems that "overlap with" is more commonly used. [Ref](https://forum.wordreference.com/threads/overlap-vs-overlap-with.248347/)~

```suggestion
- If Upgrade-A-B overlaps with Upgrade-B-C (e.g. different changes of a same configuration item), it is recommended to perform Upgrade-A-B to V-B and then perform Upgrade-B-C to V-C.
```